scripts/build/debian/config.json

```diff
@@ -4,7 +4,7 @@
   "genericName": "Etcher",
   "homepage": "https://etcher.io/",
   "icon": "assets/icon.png",
-  "bin": "etcher-electron.sh",
+  "bin": "etcher",
```

Comment:
(AFAICT) This now means that both the GUI and CLI versions of Etcher are named `etcher`? :-/